core/store/fs_directory.cpp

```diff
@@ -193,6 +201,7 @@ class fs_index_output : public buffered_index_output {
   size_t CloseImpl() final {
     const auto size = buffered_index_output::CloseImpl();
     handle.reset(nullptr);
+    rm_.file_descriptors.Decrease(1);
```

Comment:
is handle always not null here?

Reply:
It should not. I'll put an assertion in ctor